core/auth/src/test/resources/resources/process/windows-credentials-script.bat

```diff
@@ -6,7 +6,11 @@ ECHO "AccessKeyId": "%1",
 ECHO "SecretAccessKey": "%2"
 IF NOT "%3"=="" (
     ECHO ,
-    ECHO "SessionToken": "%RANDOM"
+    IF "%3"=="RANDOM_TOKEN" (
+        ECHO "SessionToken": "%RANDOM"
```

Comment:
Change to %RANDOM%

Reply:
I agree that it is not the optimal way to structure the script.